### DISCUSSION TITLE
Microfrontend

### DISCUSSION THREAD

**Discussion by rizafahmi** [ngobrolin/web#40] [Announcements]
Untuk episode mendatang, kita akan bahas tentang microfrontend nih. Kira-kira hal apa sajakah yang perlu dibahas?

**Comment by rizafahmi** [comment]
Pertanyaan dari Fatih Kalifa: Vertical vs horizontal split (atau the right way vs the wrong way 🤪)

Cache/build optimization, load time & subsequent navigation performance)

Sumber: https://bsky.app/profile/pvey.es/post/3l7xmzxtfnc2j

**Comment by rizafahmi** [comment]
Beberapa petunjuk dari @mazipan
- Vercel microfrontend https://vercel.com/blog/how-vercel-adopted-microfrontends
- Module federation for nextjs mulai diabaikan https://x.com/Maz_Ipan/status/1852679567267541269

**Comment by rizafahmi** [comment]
Pertanyaan dari @fmgono di Threads
- Pros n Cons 
- sedalam ap keterlibatan Web Components 
- Tipe aplikasi seperti apa yang cocok untuk menggunakan microfrontend (use case) 
- cara kerja bundlingnya seperti apa

**Comment by rizafahmi** [comment]
Beberapa pertanyaan dari Instagram
- Gimana cara kerjanya hingga jadi module federation
- Microfrontend di React Native
- Kapan butuh microfrontend

**Comment by ivankristianto** [comment]
- Gimana caranya handle dependencies? Misalnya kalau mau update React 17 ke 18.
